### ISSUE TITLE
FFUpdater Crash at Huawei P40

### ISSUE THREAD

**Issue by Whatsverapple** [Tobi823/ffupdater#520]
Stacktrace:
```
java.lang.IllegalStateException: Can't find download url with regex pattern '\Qhttps://dist.torproject.org/torbrowser/\E([0-9.]{4,})\Q/tor-browser-\E[0-9.]{4,}+\Q-android-aarch64-multi.apk\E'.
	at de.marmaro.krt.ffupdater.app.impl.TorBrowser.findVersionAndDownloadUrl(TorBrowser.kt:72)
	at de.marmaro.krt.ffupdater.app.impl.TorBrowser.access$findVersionAndDownloadUrl(TorBrowser.kt:26)
	at de.marmaro.krt.ffupdater.app.impl.TorBrowser$findVersionAndDownloadUrl$1.invokeSuspend(TorBrowser.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:900)
	at android.os.Handler.dispatchMessage(Handler.java:103)
	at android.os.Looper.loop(Looper.java:219)
	at android.app.ActivityThread.main(ActivityThread.java:8668)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1109)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@7456fc, Dispatchers.Main]

```
Logs:
```
java.lang.IllegalStateException: Can't find download url with regex pattern '\Qhttps://dist.torproject.org/torbrowser/\E([0-9.]{4,})\Q/tor-browser-\E[0-9.]{4,}+\Q-android-aarch64-multi.apk\E'.
	at de.marmaro.krt.ffupdater.app.impl.TorBrowser.findVersionAndDownloadUrl(TorBrowser.kt:72)
	at de.marmaro.krt.ffupdater.app.impl.TorBrowser.access$findVersionAndDownloadUrl(TorBrowser.kt:26)
	at de.marmaro.krt.ffupdater.app.impl.TorBrowser$findVersionAndDownloadUrl$1.invokeSuspend(TorBrowser.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:900)
	at android.os.Handler.dispatchMessage(Handler.java:103)
	at android.os.Looper.loop(Looper.java:219)
	at android.app.ActivityThread.main(ActivityThread.java:8668)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1109)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@7456fc, Dispatchers.Main]

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 78.2.4 (161) release Github |
| Device | JNY-LX1 (JNY-LX1, HWJNY, JNY) |
| Manufacturer | HUAWEI (HUAWEI) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 10 (SDK: 29) |
| OS | cn-central-hcd-2a-38297be571682583793104-5b4647fbf5-dbhf4, test, release-keys, 1682590447000 |

**Comment by Tobi823** [comment]
Already fixed in version 78.2.5 or later. Please update

**Comment by Whatsverapple** [comment]
I succeed to update, now it works 👍Von meinem Telefon gesendet-------- Ursprüngliche Nachricht --------Von: Tobias Hellmann ***@***.***>Datum: Fr., 20. Okt. 2023, 19:41An: Tobi823/ffupdater ***@***.***>Cc: Whatsverapple ***@***.***>, Author ***@***.***>Betreff: Re: [Tobi823/ffupdater] FFUpdater Crash at Huawei P40 (Issue #520)
Already fixed in version 78.2.5 or later. Please update
